app/controllers/concerns/foreman/controller/environments.rb

```diff
@@ -21,7 +21,12 @@ def import_environments
     if @changed["new"].size > 0 || @changed["obsolete"].size > 0 || @changed["updated"].size > 0
       render "common/_puppetclasses_or_envs_changed"
     else
-      notice _("No changes to your environments detected")
+      notice_message = _("No changes to your environments detected")
+      if !@changed['ignored'].empty?
+        notice_message << "\n"
+        notice_message << _("Ignored Environments: %s") % @changed['ignored'].keys.join(', ')
```

Comment:
1. Lower case "environments"
2. Use `.to_sentence` instead of `join(', ')`, it's locale-aware